src/firefly/js/charts/ui/CombineChart.jsx

```diff
@@ -178,9 +182,33 @@ const CombineChartDialog = ({onComplete}) => {
     );
 };
 
+
+const CascadePlots = ({doCascading, setDoCascading}) => {
+    const paddingValidator = (val) => Validate.floatRange(-1,1,0,'Cascade Padding', val, false);
+    return (
+        <Stack spacing={1}>
+            <FormControl orientation={'horizontal'} sx={{'.MuiSwitch-root': {margin: 0}}}>
+                <FormLabel>Apply Cascading: </FormLabel>
+                <Switch checked={doCascading} onChange={(e)=>setDoCascading(e.target.checked)}/>
+            </FormControl>
```

Comment:
If a SwitchInputField isn't already present, we should add one.  It'll remove the need to handle state and populate props outside of `fields`.

Reply:
You can use `<CheckBoxInputField type='switch/>`